ras_backstage/resources/sign_in/sign_in.py

```diff
@@ -53,12 +53,28 @@ def post():
         username = message_json.get('username')
         password = message_json.get('password')
 
-        # Obviously horrible, stopgap until uaa is implemented
-        if username == current_app.config['USERNAME'] and password == current_app.config['PASSWORD']:
-            # We're assuming that uaa will return an Oauth2 token though it's almost certain that
-            # this will change once we know exactly what is being returned.
-            logger.info("Authentication successful", user=username)
-            return make_response(jsonify({"token": "1234abc"}), 201)
+        if not current_app.config.get('USE_UAA'):
+            logger.info('Retrieving sign-in details')
+            if username == current_app.config['USERNAME'] and password == current_app.config['PASSWORD']:
```

Comment:
If no username and password is provided and we're not using the dev config would this evaluate to True (`None == None and None == None`) if the config isn't set correctly?

Reply:
There's validation to make sure a user cannot send no username or password, but might be worth having a test for something like this 👍

Reply:
Form validation kicks in before you get to this point which stops you from submitting nothing

Reply:
@jcox-dev yes